website/package.json

```diff
@@ -32,11 +32,11 @@
     ]
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0",
-    "@docusaurus/plugin-client-redirects": "^2.0.0",
-    "@docusaurus/plugin-pwa": "^2.0.0",
-    "@docusaurus/preset-classic": "^2.0.0",
-    "@docusaurus/remark-plugin-npm2yarn": "^2.0.0",
+    "@docusaurus/core": "^2.4.1",
+    "@docusaurus/plugin-client-redirects": "^2.4.1",
+    "@docusaurus/plugin-pwa": "^2.4.1",
+    "@docusaurus/preset-classic": "^2.4.1",
+    "@docusaurus/remark-plugin-npm2yarn": "^2.4.1",
     "clsx": "^2.0.0",
     "docusaurus-remark-plugin-tab-blocks": "^1.2.0",
```

Comment:
Just to draw your attention, Docusaurus v3 will need `docusaurus-remark-plugin-tab-blocks@next`. That‘s a beta release and it supports MDX v2. Please ping me if something does not work.

Reply:
thanks :) will take a look soon

Reply:
Looks like it's working fine 👍

Reply:
Great! I looked around the deploy preview. All looks good.